test/docx/body-reader.tests.js

```diff
@@ -94,6 +94,17 @@ test("paragraph has justification read from paragraph properties if present", fu
     assert.deepEqual(paragraph.alignment, "center");
 });
 
+test("paragraph has indent read from paragraph properties if present (Left, Right, First Line & Hanging)", function() {
+    var indentXml = new XmlElement("w:ind", {"w:left": "720", "w:right": "720", "w:firstLine": "720", "w:hanging": "720"}, []);
+    var propertiesXml = new XmlElement("w:pPr", {}, [indentXml]);
+    var paragraphXml = new XmlElement("w:p", {}, [propertiesXml]);
+    var paragraph = readXmlElementValue(paragraphXml);
+    assert.deepEqual(paragraph.indentLeft, "720");
+    assert.deepEqual(paragraph.indentRight, "720");
+    assert.deepEqual(paragraph.indentFirstLine, "720");
+    assert.deepEqual(paragraph.indentHanging, "720");
```

Comment:
Having the same value for all indents means that one of these could be reading the wrong attribute and the test wouldn't catch it. I'd suggest creating one separate test for each property instead, and testing each in isolation.

The behaviour when the attribute is missing, and also when the `w:ind` element is missing, isn't tested, so it would be good to test that for each case as well. Since the tests for all four attributes will be similar, it's probably worth parametrizing the test, similarly to the boolean run properties.